src/CoreBundle/CronJob/CronJobScheduler.php

```diff
@@ -86,7 +86,7 @@ public function calculateCurrentPlannedExecutionDate(CronJobScheduleDataModel $s
 
         $timePassedSinceLastPlannedExecution = $now->diff($lastPlannedExecution);
 
-        if ($timePassedSinceLastPlannedExecution < $executionInterval) {
```

Comment:
Did this work before?

Or in what circumstances does this work.

Reply:
removed the check and changed the do...while into a while. This has the same affect and removes complexity.